app/assets/javascripts/discourse/app/lib/plugin-api.js

```diff
@@ -721,23 +719,50 @@ class PluginApi {
   }
 
   /**
-   * Add a new button in the options popup menu.
+   * Add a new button in the composer's toolbar options popup menu.
    *
-   * Example:
+   * @callback action
+   * @param {Object} toolbarEvent - A toolbar event object.
+   * @param {function} toolbarEvent.applySurround - Surrounds the selected text with the given text.
+   * @param {function} toolbarEvent.addText - Append the given text to the selected text in the composer.
    *
```

Comment:
I tend to leave a newline between multiline "code blocks" that are related so that it is easy to differentiate groups of related code blocks easily when reading the code.